context/AuthContext.js

```diff
@@ -1,19 +1,49 @@
-import React, { children, createContext, useState } from "react";
+import React, { createContext, useState, useEffect } from "react";
+import AsyncStorage from "@react-native-async-storage/async-storage";
 export const AuthContext = createContext();
 export const AuthProvider = ({ children }) => {
-  const [isLoding, setIsLoading] = useState(true);
+  const [isLoading, setIsLoading] = useState(true);
   const [userToken, setUserToken] = useState(null);
-  const authFinish = (token) => {
-    setUserToken(token);
+
+  const authFinish = (userUid) => {
+    setUserToken(userUid);
     setIsLoading(false);
-    console.log(token);
+    storeUserToken(userUid);
   };
+
+  const retrieveData = async () => {
```

Comment:
earlier the app was not able to store the userToken so every time the app reloaded the user was prompted to go through the onboarding part again, used asyncstorage to sore the userToken value